executor/stale_txn_test.go

```diff
@@ -1122,7 +1122,7 @@ func (s *testStaleTxnSerialSuite) TestStaleSessionQuery(c *C) {
 	tk.MustExec("insert into t10 (id) values (1)")
 	time.Sleep(2 * time.Second)
 	now := time.Now()
-	tk.MustExec(`set @@tidb_read_staleness="-1s"`)
+	tk.MustExec(`set @@tidb_read_staleness="-1"`)
```

Comment:
So the minimal unit of this session variable is second?

Reply:
right.